layers/prose/packages.el

```diff
@@ -0,0 +1,48 @@
+;;; packages.el --- prose layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Chris Ewald <chrisewald@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defconst prose-packages
+  '(flycheck))
+
+(defun prose/post-init-flycheck ()
+
+  (defun prose-proselint-enabled-p ()
+    "Workaround for proselint flycheck checker predicate"
+    prose-proselint-enabled)
+
+  (flycheck-define-checker proselint
```

Comment:
Mb it will better to add this https://github.com/amperser/proselint/blob/master/plugins/flycheck/ to the load path instead of embedding? This way we won't need to maintain the code.